packages/react/modules/components/configure-adapter/configure-adapter.spec.js

```diff
@@ -75,10 +82,11 @@ describe('lifecycle', () => {
                 <ChildComponent />
               </ConfigureAdapter>
             );
-          });
 
-          beforeEach(() => {
-            wrapper.instance().componentDidMount();
+            wrapper
+              .instance()
+              .componentDidMount()
+              .catch(() => {});
```

Comment:
Hmm, shouldn't you do `return wrapper...`? `componentDidMount` returns a Promise, so `beforeEach` should wait for it to be resolved I suppose, or?

Reply:
True. I guess it just happened to work.